source/renderer/UiRenderer.cpp

```diff
@@ -50,7 +50,7 @@ void UiRenderer::render() {
   Text version;
   version.font     = "Pacaya";
   version.size     = 0.5f;
-  version.content  = std::string("Early testing build: ")  + GAME_VERSION;
+  version.content  = std::string("Early testing build: "); //+ GAME_VERSION; // For windows does not work
```

Comment:
this is intended function / feature
it is saved in version.hpp and is generated with cmake target versionInfo
i think this file is already initielized in the repository and i think it will stay that way
the best solution should be to improve cmakelist.txt to dont generate this file with msvs compiler